src/core/Address.ts

```diff
@@ -74,9 +74,9 @@ export abstract class Address {
             }
             try {
                 const address = SymbolAddress.createFromString(encodedAddress);
-                const hasher = new SHA3(256);
+                const hasher = sha3_256.create();
                 const hash = hasher.update(Buffer.from(address.rawAddress.addressWithoutChecksum)).digest();
```

Comment:
it looks like `address.rawAddress.addressWithoutChecksum` is already a Uint8Array, so you shouldn't need to wrap it in a Buffer. 

notice also that this  `Buffer.from` overload is making a copy, which is undesirably. where possible, you should be using the ArrayBuffer overload [`Buffer.from(..addressWithoutChecksum.buffer)`] that only creates a view and not a memory copy.

Reply:
also, you probably want to call `arrayBuffer` instead of `digest`.  the former returns an `ArrayBuffer`, which when wrapped by Uint8Array only creates a view and not a memcpy.